### ISSUE TITLE
Modifica intestazione privacy policy

### ISSUE THREAD

**Issue by brunosaetta** [FattiDiCookies/italianPrivacyPolicy#66]
1) E' rivolta agli italiani, quindi propongo di utilizzare il più possibile la lingua italiana. Privacy policy non si capisce, informativa è chiaro. Quindi: 

"INFORMATIVA PRIVACY (trattamento dei dati personali)"

2) La prima informazione che occorre all'utente è capire a che serve l'informativa, e la risposta è che serve per tutelare i suoi diritti perchè lo prevede la legge (e non che ti dico come tratti i tuoi dati perchè poi l'utente si chiede per quale motivo dovrebbe darmeli i dati). 

"In ottemperanza degli obblighi derivanti dalla normativa nazionale e comunitaria in materia di trattamenti dei dati personali, il sito [[NOME SITO]] rispetta e tutela la riservatezza dei visitatori e degli utenti. "

3) Molti siti sono legati a realtà fisiche, aziende ecc... Occorre precisare se l'informativa riguarda tutto il complesso dei dati oppure no: 
"La presente informativa si applica esclusivamente alle attività online del presente sito, non si applica alle informazioni raccolte tramite canali diversi. "

Proposta (da sostiture all'intestazione e primo paragrafo): 

INFORMATIVA PRIVACY e COOKIE (trattamento dei dati personali)

In ottemperanza degli obblighi derivanti dalla normativa nazionale e comunitaria in materia di trattamenti dei dati personali, il sito [[NOME SITO]] rispetta e tutela la riservatezza dei visitatori e degli utenti. 
La presente informativa si applica esclusivamente alle attività online del presente sito, non si applica alle informazioni raccolte tramite canali diversi.

**Comment by Gix075** [comment]
Per me va benissimo, se non ci sono pareri discordanti entro breve, domani apporto la modifica al testo ufficiale.

**Comment by Gix075** [comment]
Visto che nessuno ha posto obiezioni alla modifica proposta da @brunosaetta la integro nel testo ufficiale.

@brunosaetta ho inserito il testo in https://github.com/FattiDiCookies/italianPrivacyPolicy/blob/master/docs/privacy-policy/md/privacy-policy.md 
spero di aver capito bene altrimenti dimmelo che lo correggo ancora.

**Comment by iusondemand** [comment]
Scusate la lentezza.
Sul punto tre ho dei dubbi. @brunosaetta a volte la policy online e' l'unica occasione di gestire in un unico posto tutti i trattamenti che possono partire dal sito e continuare in locale. In altri casi invece e' bene separare tutto. Non sarebbe meglio fare attenzione all'ambito nel momento in cui si descrive il trattamento ? Alcuni trattamenti restano sul sito, altri finiscono altrove (Commercialista, fatturazione, spedizioniere ...)

**Comment by brunosaetta** [comment]
Certo @iusondemand , ma l'idea era di fare una privacy semplice (ma completa), non per aziende, che necessitano di personalizzazioni specifiche. . 
Comunque per la precisione il punto 3 riguarda tutti i dati raccolti tramite il sito, anche se poi vanno inviati a terzi per ulteriori elaborazioni (questo lo chiarirò nella parte sulle finalità -da riscrivere-). Sono esclusi solo i dati raccolti al di fuori dal sito (tipo moduli cartacei, telefono, ecc....)

**Comment by Gix075** [comment]
@brunosaetta anche secondo me, almeno in prima battuta, va fatta una policy semplice che si possa adattare a tutte le situazioni più generiche. 

Per quello che riguarda le finalità del trattamento però dovremmo pensarle in maniera modulare in modo che con il tool possano essere composte dinamicamente a seconda delle esigenze perché immagino siano diverse le finalità da chi ha solo un modulo di contatto a chi fa invece invii massicci di mail pubblicitarie e via dicendo. Dico bene?

**Comment by brunosaetta** [comment]
@Gix075 giusto, infatti le finalità che considero sono: statistica; sicurezza; marketing; shopping, performance; attività accessorie. Poi vediamo come organizzare il tutto.
